### DISCUSSION TITLE
It is possible to made tablet version?

### DISCUSSION THREAD

**Discussion by yaoleynikov** [MustardChef/WSABuilds#212] [Ideas]
It is possible to made tablet version? Because on 4k monitor, apps look very stretched.

**Comment by MustardChef** [comment]
I don't develop WSA, nor do I claim to. This is an unofficial project.
The best places to give feedback to, are to the app developers themselves.
Also, if this is present in all apps, you can use the Feedback Hub (Windows Key + F) and report this to the WSA team.

**Comment by Mitia99** [comment]
@yaoleynikov I have a Windows tablet and WSA auto use tablet app UI. Any screenshot of your issue?
